README.md

```diff
@@ -10,22 +10,29 @@
 | NPM | 6.13.4 | https://docs.npmjs.com/cli/v7/commands/npm-install | Software Package Manager |
 | VS Code | 6.13.4 | https://code.visualstudio.com/ | Code IDE, not required but popular choice for this project |
 
-#### Starting Application Locally
-First, the following environment variables, `REACT_APP_MAPBOX_API_KEY` and `REACT_APP_ROUTE` must be set. Talk to a team member to get the keys.
+#### Installing Node & NPM 
 
-Second, install npm packages:
-```bash
-npm install
-```
-*Note: This needs to be run when you initially setup the repository and when packages are updated/added/removed.*
+We recommend using Node Version Manager (nvm), which is a command line tool that allows you to quickly install, use and switch between different versions of node. If you're using a bash shell (default shell for Mac OS), simply execute the following steps:
```

Comment:
Linking to these instructions for Windows users would be good https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows

Reply:
These instructions should work for windows users as well according to https://github.com/nvm-sh/nvm#about

Reply:
I'm having some performance issues running it on Windows natively though, so I'll probably install it using WSL eventually. When I do, I can document any issues I have and add them here.

Reply:
Installed it in WSL and the performance difference is so drastic that I would almost avoid mentioning the native Windows installation. Native Windows was almost unusable, WSL is extremely quick.